theano/tensor/nnet/tests/test_neighbours.py

```diff
@@ -283,9 +283,9 @@ def test_neibs_full_step_by_valid(self):
                     padded_x = T.set_subtensor(padded_x[:, :, extra[0]:-extra[0], extra[1]:-extra[1]], x)
                     x_using_valid = images2neibs(padded_x, neib_shape, neib_step, mode="valid")
                     x_using_full = images2neibs(x, neib_shape, neib_step, mode="full")
-                    close = T.allclose(x_using_valid, x_using_full)
-                    f = theano.function([], close, mode=self.mode)
-                    assert f()
+                    f_valid = theano.function([], x_using_valid, mode='FAST_RUN')
+                    f_full = theano.function([], x_using_full, mode=self.mode)
+                    unittest_tools.assert_allclose(f_valid(), f_full())
```

Comment:
Did you time that this make a difference in the test time? This lower a little bit the test coverage as DebugMode isn't used. But I think think this is a problem. If you see test time difference on your computer, I'm ok to merge this PR.